inc/class-shortcake-bakery.php

```diff
@@ -139,6 +140,13 @@ public function action_admin_enqueue_scripts() {
 			'nonces' => array(
 				'customEmbedReverse' => wp_create_nonce( 'embed-reverse' ),
 			),
+			'shortcodes' => array_flip(
+				array_map(
+					function( $class ) {
+						return str_replace( 'Shortcake_Bakery\\Shortcodes\\', '', $class );
```

Comment:
This whole point here to to pass the shortcode tag into javascript; it passes an object that looks like `"GoogleDocs": "googledocs"` or whatever the tag names are filtered to - ie `"GoogleDocs": "fusion-googledocs"`...

I guess the `str_replace` isn't necessary, other than making the js object a little less messy looking. It'd work just as well to pass the full namespaced class as the keys of the object. I'll do that.